jquery.jscroll.js

```diff
@@ -8,8 +8,16 @@
  * @version 2.3.7
  * @requires jQuery v1.4.3+
  * @preserve
+ * @Supports AMD pattern
  */
-(function($) {
+(function($, factory){
+    'use strict';
+    if (typeof define === 'function' && define.amd) {
+        return factory($);
+    } else {
+        window.jScroll = factory($);
```

Comment:
I think this is not needed, because it's a jquery plugin and is therefore attached to jquery, see https://github.com/pklauzinski/jscroll/blob/master/jquery.jscroll.js#L214